// jet/jet.js
// passive view mvc - all it can do is render models, strings, and other data
/*
// config
var Todos = new App.Controller();

// model logic
Todos.Model.create({
    items: [],
    item: function() {
        Todos.keyTrigger.call()
    }
});

// view logic
Todos.View.create({
    main: "#main",
    item: "#item",
    list: "#list",
    footer: "#footer"
});

// controller logic
Todos.createEvent({
    keyTrigger: function() {
        $("#text").keyup(function(event){
            if(event.keyCode == 13) {
                Todos.Model.items.push($("#text").val());
            }
        });
    }
});

Todos.main.on({
    init: function() {
        Todos.item.on({
            "itemName": function() {} // do something with all these empty functions!!!
        });
        Todos.list.on({
            "items": function() {}
        });
        Todos.footer.on({
            "itemCount": function() {}
        });
    }
});

*/
exports.use = function(file) {
	return require(file);  // use the fs module
};

exports.jet = {
    use: function(str) {
    	console.log(str);

		this.Controller = function() {
			this.talk = function() {
				console.log("hello");
			};
		};

    }
};
/*
var App = new Jet.use("./hello");
var Todos = new App.Controller();
Todos.talk();
*/
